### DISCUSSION TITLE
Random repository wrongly declaring to use Fprime as a dependency

### DISCUSSION THREAD

**Discussion by SMorettini** [nasa/fprime#2265] [General]
I just noticed that there are some repositories that say to have Fprime as a dependency but actually, they just put the link to this repository in their repository or by mistake to reach more coverage  https://github.com/nasa/fprime/network/dependents?package_id=UGFja2FnZS0yMjkwNDQxMDAx

**Comment by Joshua-Anderson** [comment]
Not sure there's anything actionable we can do here unfortunately. Github seems to not have fully considered the abuse possibilities of the feature.
